<launch>
    <node pkg="prometheus_detection" type="oneshot_detection_client.py" name="oneshot_detection_client" output="screen">
        <param name="output_topic" value="/prometheus/object_detection/oneshot_det"/>
        <param name="camera_parameters" type="string" value="$(find prometheus_detection)/shell/calib_webcam_640x480.yaml" />
        <param name="object_name" type="string" value="object" />
        <param name="object_height" type="double" value="0.2" />
    </node>
</launch>
